internal/docker/builder.go

```diff
@@ -351,11 +346,19 @@ func (d *Builder) construct(bprint b.Blueprint) (errs []error) {
 	return errs
 }
 
+func toChanges(labels map[string]string) []string {
+	changes := make([]string, 0)
+	for k, v := range labels {
+		changes = append(changes, fmt.Sprintf("LABEL \"%s\"=\"%s\"", k, v))
```

Comment:
Please document assumptions you're making here. Not all characters will be supported in this format, no? Which ones would be problematic?